eng/docker/build-docker-sdk.ps1

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env pwsh
+# Builds libraries and produces a dotnet sdk docker image
+# that contains the current bits in its shared framework folder.
+
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string][Alias('t')]$imageName = "dotnet-sdk-libs-current",
+  [string][Alias('c')]$configuration = "Release",
+  [switch][Alias('w')]$buildWindowsContainers
+)
+
+$REPO_ROOT_DIR=$(git -C "$PSScriptRoot" rev-parse --show-toplevel)
```

Comment:
Should we remove the git dependency here at some point?

Reply:
And replace with a relative path? Do we expect this to run outside of a repo?